tests/ui/suggestions/crate-or-module-typo.stderr

```diff
@@ -41,7 +41,7 @@ LL -     bar: st::cell::Cell<bool>
 LL +     bar: cell::Cell<bool>
    |
 
-error[E0433]: failed to resolve: function `bar` is not a crate or module
+error[E0433]: cannot find item `bar` in this scope
```

Comment:
You can find `bar`, but it's a function and not a module, the previous error told about that.